packages/ui-bindings/Unit/getUnitById.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * A method to get a single Unit
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2017-12-04
+ */
+
+import { connect } from 'react-redux'
+import { gql, graphql, compose} from 'react-apollo'
+
+import { AppState } from '@vflows/store/types'
+import { getActiveLoginToken } from '@vflows/store/selectors/auth'
+
+import { unitInterface } from "./unit";
+
+const query = gql`
+query($token: String, $UnitId: Int) {
+  viewer(token: $token) {
+    unit(id: $UnitId){
+      ...unitInterface
+    }
+  }
+}
+${unitInterface}
+`
+
+export default compose(
+  // bind input data from the store
+  connect((state: AppState) => ({
+    variables: {
+      token: getActiveLoginToken(state),
+    },
+  })),
+
+  graphql(query, {
+    // read query vars into query from input data above
+    options: (props) => (
```

Comment:
None of these statements are aligned. It makes it more difficult to read what is embedded in what. Please consider aligning. In JetBrains IDEs, CTRL + A, CTRL + ALT + i

Reply:
resolved